cmd/configtxgen/main_test.go

```diff
@@ -77,7 +77,19 @@ func TestMissingConsortiumValue(t *testing.T) {
 	config := genesisconfig.Load(genesisconfig.SampleSingleMSPChannelProfile, configtest.GetDevConfigDir())
 	config.Consortium = ""
 
-	assert.Error(t, doOutputChannelCreateTx(config, nil, "foo", configTxDest), "Missing Consortium value in Application Profile definition")
+	assert.EqualError(t, doOutputChannelCreateTx(config, nil, "foo", configTxDest), "config update generation failure: cannot define a new channel with no Consortium value")
+}
+
+func TestUnsuccessfulChannelTxFileCreation(t *testing.T) {
+	configTxDest := filepath.Join(tmpDir, "configtx")
+
+	config := genesisconfig.Load(genesisconfig.SampleSingleMSPChannelProfile, configtest.GetDevConfigDir())
+	assert.NoError(t, ioutil.WriteFile(configTxDest, []byte{}, 0440))
+	defer os.Remove(configTxDest)
+
+	err := doOutputChannelCreateTx(config, nil, "foo", configTxDest).Error()
+	assert.Error(t, doOutputChannelCreateTx(config, nil, "foo", configTxDest), "Not able to write to file")
+	assert.Regexp(t, "Error writing channel create tx: open .+/configtxgen.+/configtx: permission denied", err)
```

Comment:
```suggestion
	assert.EqualError(t, doOutputChannelCreateTx(config, nil, "foo", configTxDest), fmt.Sprintf("Error writing channel create tx: open %s: permission denied", configTxDest))
```
Not trying to stall on this one, but considering I'm not a maintainer and can't merge this in anyways, can't we simplify this down to just one line and avoid the regular expression by formatting in the path since we have the variable with the exact path. Tried this out locally and it works for me. Seems more flexible to me in case somebody down the line decided to change the name of the tmpDir.

Reply:
Sure, sure, I totally got your point. Pushing a change now. Let me know about the new version.